### ISSUE TITLE
It should be easy to expand a search cover all resources, rather than a single file

### ISSUE THREAD

**Issue by eemeli** [mozilla/pontoon#3037]
The navigation in Pontoon's translate view most naturally guides users to seeing a string within its context of the file/resource in which it's stored. This is mostly good, but it has a side effect of the string search being filtered by the current resource, even when its placeholder says "Search in All".

We should:

- [ ] Clarify when the search is so filtered, e.g. with "Search in this resource" as the text.
- [ ] Provide for an easy way to expand a search to cover all resources, and possibly even all projects.

**Comment by mathjazz** [comment]
> Clarify when the search is so filtered, e.g. with "Search in this resource" as the text.

"Search in All" is indeed misleading. It refers to searching among all strings in the selected path, because none of the filters is applied.

>  Provide for an easy way to expand a search to cover all resources, and possibly even all projects.

You can do so  by switching to the All Resources / All Projects view or by using Concordance Search.

**Comment by eemeli** [comment]
> > Provide for an easy way to expand a search to cover all resources, and possibly even all projects.
> 
> You can do so by switching to the All Resources / All Projects view or by using Concordance Search.

Yes, but we don't have a way of a user to change the resource/project scope of their search, the way that they can change e.g. translation status, submission time or author filters. The current methods all require repeating the search in a different context.

**Comment by eemeli** [comment]
That might be one possible solution, yes. But I rather explicitly didn't include a solution in the description, because there's more than one possibility here. Additional ones that come to my mind:
- Include a link "Expand search to all resources in _Project_" at the end of the search results.
- Include some button in the filter dialog for expanding the search to all resources or even all projects.

I couldn't say right away what might be best, so I left that out. The design needs to be a part of the work here.
